src/map/clif.cpp

```diff
@@ -5242,6 +5244,12 @@ int clif_damage(struct block_list* src, struct block_list* dst, t_tick tick, int
 		}
 	}
 
+	// Calculate what sdelay to send to the client so it applies damage at the same time as the server
+	if (battle_config.synchronize_damage && src->type == BL_MOB && status_get_clientamotion(src) != 0) {
```

Comment:
status_get_clientamotion will always return != 0, since you cap it to 1

Reply:
Should still check for it since we are making a division.

Reply:
Then put std::max into the formula, if you want to be safe.
But not checking here for something that cannot happen and hiding the actual reason behind the check (division by zero safeguard vs value not set).

Reply:
Okay can do that.